posthog/rate_limit.py

```diff
@@ -24,3 +24,13 @@ class PassThroughBurstRateThrottle(PassThroughThrottle):
 
 class PassThroughSustainedRateThrottle(PassThroughThrottle):
     scope = "sustained"
+
+
+class DestroyClickhouseModelThrottle(UserRateThrottle):
+    rate = "10/hour"
```

Comment:
Is this delete 10 people per hour? If so that seems overly harsh....

Reply:
See PR description - yes, it's harsh, but this is currently causing a lot of pain on cloud.

Reply:
Is it worth making the rate into an instance setting so we can change it easily if it's _too_ harsh?

Reply:
Note that rate limits are only applied on cloud right now.